cmd/jaeger/config.yaml

```diff
@@ -18,8 +18,13 @@ service:
                 port: 8888
     logs:
       level: debug
-    # TODO Initialize telemetry tracer once OTEL released new feature.
-    # https://github.com/open-telemetry/opentelemetry-collector/issues/10663
+    traces:
```

Comment:
we don't this by default in this config, but in cmd/jaeger/internal/all-in-one.yaml